cyclegan_arch/mnist_to_svhn/data_loader.py

```diff
@@ -6,9 +6,9 @@ def get_loader(config):
     """Builds and returns Dataloader for MNIST and SVHN dataset."""
     
     transform = transforms.Compose([
-                    transforms.Scale(config.image_size),
+                    transforms.Resize(config.image_size),
                     transforms.ToTensor(),
-                    transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
```

Comment:
Throws error when using this transform on MNIST which is grayscale and this function expected RGB.
Changed to handle both RGB and Grayscale